dataland-frontend/src/components/resources/dataTable/MultiLayerDataTableBody.vue

```diff
@@ -3,11 +3,10 @@
     <template v-if="isCellOrSectionVisible(cellOrSectionConfig, mldtDatasets)">
       <tr
         v-if="cellOrSectionConfig.type == 'cell'"
-        v-show="isVisible"
-        :class="cellOrSectionConfig.class ?? null"
```

Comment:
Why would you remove this class here?

Reply:
@marcelrasbachdfine do you know something about the css classes? I do not recall changing them

Reply:
no i dont. only intended thing was to extend the v-show condition

Reply:
Maybe that got merged in. Let me check.

Reply:
Should be back